docker-compose.yml

```diff
@@ -147,3 +147,13 @@ services:
       - "8090:8090"
     environment:
       DATABASE_CONN: 'postgres+psycopg2://deploy:deploy@db/analytics'
+
+  nginx:
+    image: nginx:1.17.9
```

Comment:
We don't want to add `nginx` to the docker-compose file here; the PR should only add a `/v1/thumbs` endpoint that resolves a URL to a proxied image.